rally_ovs/plugins/ovs/scenarios/ovn_network.py

```diff
@@ -67,6 +67,10 @@ def create_routers_bind_ports(self, router_create_args=None,
 
         # Create ports on the logical networks
         sandboxes = self.context["sandboxes"]
+        if not sandboxes:
+            # We need to get all the sandbox on all farms for random
```

Comment:
The comment is misleading. Whether or not to do random distribution doesn't rely on this condition "if not sandboxes". The change here is about "when there is no sandbox specified, we bind ports on all sandboxes". (note that, even when using all sandboxes, it still can be even distribution if number of lports is bigger than number of sandboxes.

Reply:
on L74, the warning "Number of ports less than chassis: inbalance binding\n" should be updated to:
"Number of ports less than chassis: random binding\n"

Reply:
Sure will correct the comments and message.